tests/system/small/test_unordered.py

```diff
@@ -26,3 +27,34 @@ def test_unordered_mode_cache_aggregate(unordered_session):
     pd_result = pd_df - pd_df.mean()
 
     assert_pandas_df_equal(bf_result, pd_result, ignore_order=True)
+
+
+@skip_legacy_pandas
+def test_unordered_mode_read_gbq(unordered_session):
+    df = unordered_session.read_gbq(
```

Comment:
can the test be flaky if the `unordered_session` may return data in a different order?

Reply:
yes, should have `ignore_order=True` like the other test. added